modules/extensions-support/src/test/java/org/mule/module/extension/internal/metadata/extension/MetadataOperations.java

```diff
@@ -111,6 +112,26 @@ public Object outputMetadataWithoutKeysWithKeyParam(@Connection MetadataConnecti
         return null;
     }
 
+    // Resolver for output and content
+    // Without keysResolver
+    // With KeyParam
+    // Uses Cache
+    @MetadataScope(outputResolver = TestResolverWithCache.class, contentResolver = TestResolverWithCache.class)
+    public Object contentAndOutputCacheResolver(@Connection MetadataConnection connection, @MetadataKeyParam String type, @Content Object content)
+    {
+        return null;
+    }
+
+    // Resolver for content
```

Comment:
why 4 lines for the comment? Why the exact same comment for the two methods? Doesn't it makes sense to have a 1 line javadoc at the class level?